virttest/qemu_vm.py

```diff
@@ -1371,9 +1371,11 @@ def sort_key(dev):
 
         self.qemu_binary = qemu_binary
         self.qemu_version = process.system_output("%s -version" % qemu_binary,
+                                                  verbose=False,
```

Comment:
why disable version info?

Reply:
We have it at the preprocess phase.
This one's output is 
06:09:47 INFO | Running '/sbin/ovs-vswitchd --version'
06:09:47 DEBUG| [stdout] ovs-vswitchd (Open vSwitch) 2.0.0
06:09:47 INFO | Command '/sbin/ovs-vswitchd --version' finished with 0 after 0.0038321018219s
06:09:47 DEBUG| [stdout] Compiled Jan 26 2014 14:22:04
06:09:47 DEBUG| [stdout] OpenFlow versions 0x1:0x1
06:09:47 INFO | Running '/sbin/ovs-vswitchd --version'
06:09:47 DEBUG| [stdout] ovs-vswitchd (Open vSwitch) 2.0.0
06:09:47 INFO | Command '/sbin/ovs-vswitchd --version' finished with 0 after 0.0037670135498s
06:09:47 DEBUG| [stdout] Compiled Jan 26 2014 14:22:04